libs/backend/src/app.ts

```diff
@@ -33,6 +39,25 @@ server.register(cors);
 server.register(swagger);
 server.register(jwt);
 server.register(swaggerUi);
+server.register(websocket);
+
+const games: WebSocketGamesMap = new Map();
+const activePlayerList: WebSocket[] = [];
+
+server.register(async function (fastify) {
+	fastify.get("/", { websocket: true }, (socket: WebSocket, req: FastifyRequest) => {
+		onConnection({ activePlayers: activePlayerList, games, newPlayerSocket: socket });
+
+		socket.on("message", (message) => {
+			onMessage({ message, games, socket, activePlayers: activePlayerList });
+			updateActivePlayers({ sockets: activePlayerList, games });
+		});
+
+		socket.on("close", (code, reason) => {
+			onClose({ code, reason, activePlayers: activePlayerList, playerSocketToRemove: socket });
+		});
+	});
+});
```

Comment:
preferably handle this in a different file all-together and just import/execute the file here, separation of concerns :+1:

Reply:
would make it so this file is easier to maintain => only imports/setting up stuff for fastify => doesn't belong here, already actual logic